### PR TITLE
Fixes/#267-finish-eGon100RE-power-plants

### LINKED ISSUE #267: finish-eGon100RE-power-plants fails
Several functions in the power-plants dataset fail when scenario is eGon100RE

### PR DESCRIPTION
**Pull request by CarlosEpia**
Fixes #267.

### CODE REVIEW

File: src/egon/data/datasets/power_plants/__init__.py [hunk 1]
```diff
@@ -237,8 +237,11 @@ def insert_biomass_plants(scenario):
     """
     cfg = egon.data.config.datasets()["power_plants"]
 
-    # import target values from NEP 2021, scneario C 2035
-    target = select_target("biomass", scenario)
+    # import target values
+    if scenario != "eGon100RE":
+        target = select_target("biomass", scenario)
+    else:
+        target = 20000  # dummy value. Muss be fix with issue #268
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/power_plants/__init__.py]: I just saw your question in the referenced issue and commented there. We will not have biomass in this scenario.

File: src/egon/data/datasets/power_plants/wind_farms.py [hunk 2]
```diff
@@ -313,7 +326,9 @@ def match_district_se(x):
     if fed_state in north:
         state_wf["inst capacity [MW]"] = power_north * state_wf["area [km²]"]
     else:
-        state_wf["inst capacity [MW]"] = power_south * state_wf["area [km²]"]
+        state_wf["inst capacity [MW]"] = (
+            power_south * state_wf["area [km²]"]
+        )  # ToDo: fix this when scn eGon100RE
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/power_plants/wind_farms.py]: Is this still an open todo?

File: src/egon/data/datasets/power_plants/wind_farms.py [hunk 3]
```diff
@@ -79,34 +79,49 @@ def insert():
...
         )
-        summary_t = pd.concat([summary_t, summary_state])
-        farms = pd.concat([farms, wind_farms_state])
+        target_power_df = target_power_df[
+            target_power_df["scenario_name"] != "eGon100RE"
+        ]
+
+    if "eGon2035" in target_power_df["scenario_name"].values:
+        # Fit wind farms scenarions for each one of the states
+        for scenario in target_power_df.index:
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/power_plants/wind_farms.py]: I am wondering if this loop is needed when eGon100RE was already created in the upper function calls.

File: src/egon/data/datasets/power_plants/__init__.py (hunk 1 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/power_plants/__init__.py]: Changes applied in 0c9b280a4c4f2682db9e7587e133a97be377593b

File: src/egon/data/datasets/scenario_parameters/parameters.py [hunk 4]
```diff
@@ -105,6 +105,7 @@ def global_settings(scenario):
                 "coal": 20.2,  # [EUR/MWh]
                 "lignite": 4.0,  # [EUR/MWh]
                 "nuclear": 1.7,  # [EUR/MWh]
+                "biomass": 40,  # Dummyvalue, ToDo: Find a suitable source. To be fix in #268
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/scenario_parameters/parameters.py]: This is not needed anymore, right?

File: src/egon/data/datasets/power_plants/wind_farms.py (hunk 2 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/power_plants/wind_farms.py]: Nice that you mentioned it. This ToDo is now fixed in a0dc3fd7987bd31b82b42c2463fe181bf2263d0d

File: src/egon/data/datasets/power_plants/wind_farms.py (hunk 3 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/power_plants/wind_farms.py]: It is a process that is performed by federal state, so the for loop is necessary. I think the variable name is misleading, so I changed it in b3f87c8ff842201f20e90fe95bdba676ab060c13

File: src/egon/data/datasets/scenario_parameters/parameters.py (hunk 4 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/scenario_parameters/parameters.py]: Right. It was deleted in dd2488fe2470a489ba8f4fda8c5df79b03d4c410.

**Review by ClaraBuettner — APPROVED** [review]

**Pull request merged**